firmware/application/apps/capture_app.cpp

```diff
@@ -60,25 +60,27 @@ CaptureAppView::CaptureAppView(NavigationView& nav)
     };
 
     freqman_set_bandwidth_option(SPEC_MODULATION, option_bandwidth);
-    option_bandwidth.on_change = [this](size_t, uint32_t base_rate) {
-        /* base_rate is used for FFT calculation and display LCD, and also in recording writing SD Card rate. */
+    option_bandwidth.on_change = [this](size_t, uint32_t bandwidth) {
+        /* bandwidth is used for FFT calculation and display LCD, and also in recording writing SD Card rate. */
```

Comment:
I would leave in the comments the word of sample rate when talking about base clock
but it is true that  SR = Bandwith (in SDR , when getting 2 x samples by clock , I,Q)